README.md

```diff
@@ -74,4 +77,11 @@ we refer to collectively as "extensions") in the
 [Kong Hub](https://docs.konghq.com/hub), with documentation hosted
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
+## Contributing 
+We :heart: contributions from everyone! We welcome [bug reports](https://github.com/Kong/docs.konghq.com/issues/), [bug fixes](https://github.com/Kong/docs.konghq.com/pulls) and feedback on the library is always appreciated. Look at the [Contributor Guidelines](CONTRIBUTING.md) for more information.
```

Comment:
"feedback on the library" - well, there isn't a "library" per se. Also, feedback should be directed to https://discuss.konghq.com/